docs/6.x/cluster.md

```diff
@@ -4,12 +4,12 @@ Every Cluster created from a Gravity Cluster Image is a fully featured
 Kubernetes environment. It contains the following components:
 
 1. All user applications that have been packaged into the Cluster Image.
-2. All Kubernetes daemons like `kube-scheduler`, `kube-apiserver`, and others. 
```

Comment:
That's just my text editor auto-truncating trailing whitespaces. There's a bunch of these here.

Reply:
😡

Reply:
@benarent FWIW I think everyone should configure their editors to truncate trailing whitespaces :)